sdk/rpc/server/server.go

```diff
@@ -136,17 +137,18 @@ func (t *RPCServer) CredentialProvisionerDescription(req proto.ProvisionerID, re
 // interface. The call is forwarded to the Provision() function of the Provisioner of the credential identified by
 // req.CredentialID.
 func (t *RPCServer) CredentialProvisionerProvision(req proto.ProvisionCredentialRequest, resp *sdk.ProvisionOutput) error {
+	defer func() {
+		if err := recover(); err != nil {
+			resp.AddError(fmt.Errorf("your locally built plugin failed with the following panic: %s; and with stack trace: %s", err, string(debug.Stack())))
```

Comment:
I really like that we're improving what happens on a panic. But could we extract this into a separate PR? I'd like to have a discussion about the consistency between calls and whether we want to use `resp.AddError()` for this. I think there is something to say for having a special type in the `proto` package to encode panics.

Reply:
Sure thing, I'll extract this in another PR.